allennlp/models/model.py

```diff
@@ -87,8 +101,70 @@ def get_metrics(self, reset: bool = False) -> Dict[str, float]:
         # pylint: disable=unused-argument,no-self-use
         return {}
 
+    def archive(self,                                       # pylint: disable=no-self-use
```

Comment:
What do you think about these be separate functions in a `model_utils.py` or something? A research user doesn't need to know about this and it unnecessarily clutters the `Model` API.